app/server/services/sendgrid_email.js

```diff
@@ -0,0 +1,115 @@
+ * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {Object}           templateData  [Mapping of template dynamic fields to values]
+ * @param  {Function}         callback      [Callback to be executed on success or error]
+ */
+function sendOne(template, email, templateData, callback) {
+  var msg = {
+    to: email,
+    from: EMAIL_ADDRESS,
+    templateId: template,
+    dynamic_template_data: templateData
+  };
+  sgMail.send(msg, (err, info) => {
+    if (err) {
+      console.error('Error sending email through SendGrid: ' + err);
+    }
+    if (info) {
+      console.warn('Email with template ' + template + ' sent to ' + email + ' with response:\n');
+      if (info[0] && info[0].toJSON) {
+        console.warn(info[0].toJSON());
+      }
+      else {
+        console.error("Ill-formed SendGrid response log"); 
+      }
+    }
+    if (callback) {
+      callback(err, info);
+    }
+  });
+}
+
+/**
+ * Send a verification email to a user, with a verification token to enter.
+ * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {[type]}     token    [description]
+ * @param  {Function} callback [description]
+ * @return {[type]}            [description]
+ */
+emailService.sendVerificationEmail = function (email, token, callback) {
+  var verifyUrlData = {
+    'url': ROOT_URL + '/verify/' + token
+  };
+  sendOne(VERIFICATION_EMAIL_TEMPLATE, email, verifyUrlData, callback);
+};
+
+/**
+ * Send a password recovery email.
+ * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {[type]}   token    [description]
+ * @param  {Function} callback [description]
+ */
+emailService.sendPasswordResetEmail = function (email, token, callback) {
+  var resetUrlData = {
+    'url': ROOT_URL + '/reset/' + token
+  }
+  sendOne(PASSWORD_RESET_EMAIL_TEMPLATE, email, resetUrlData, callback);
+};
+
+/**
+ * Send a password recovery email.
+ * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {Function} callback [description]
+ */
+emailService.sendPasswordChangedEmail = function (email, callback) {
+  sendOne(PASSWORD_CHANGED_EMAIL_TEMPLATE, email, {}, callback);
+};
+
+/**
+ * Send the acceptance email to the participant.
+ * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {[type]}   confirmBy [description]
+ * @param  {Function} callback  [description]
+ */
+emailService.sendAcceptanceEmail = function (email, confirmBy, callback) {
+  confirmDate = { 
+    'date': moment(confirmBy).format('MMMM D, YYYY h:mm A') 
+  };
+  sendOne(ACCEPTANCE_EMAIL_TEMPLATE, email, confirmDate, callback);
+};
+
+/**
+ * Send the waiver email to the participant.
+ * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {Function} callback  [description]
+ */
+emailService.sendWaiverEmail = function (email, callback) {
+  var userEmailData = {
+    'email': email
```

Comment:
```suggestion
    email
```

Reference: https://javascript.info/object#property-value-shorthand